.github/workflows/integration-test.yml

```diff
@@ -87,5 +87,5 @@ jobs:
         run: |
           echo "check if opensearch is ready"
           curl -XGET https://localhost:9200 -u 'admin:admin' -k
-          node .\test\run_jest_tests.js --config .\test\jest.config.server.js --testPathIgnorePatterns saml_auth.test.ts
+          node .\test\run_jest_tests.js --config .\test\jest.config.server.js --testPathIgnorePatterns saml_auth.test.ts --testPathIgnorePatterns saml_multiauth.test.ts
```

Comment:
What is this change doing?

Reply:
Ignoring the test on windows. I'll resurrect the discussion where this was disabled for saml integ tests w/o multiauth

Reply:
Seem @RyanL1997's comments here: https://github.com/opensearch-project/security-dashboards-plugin/pull/1275#issuecomment-1374256972

Reply:
Btw just wanna confirm, are we not ahving the old `saml_auth.test.ts` any more?